src/components/project/project.resolver.ts

```diff
@@ -254,6 +258,20 @@ export class ProjectResolver {
     return list;
   }
 
+  @ResolveField(() => Partnership)
+  async partnership(
+    @Parent() project: Project,
+    @IdArg({ name: 'partner' }) partnerId: ID,
+    @Loader(PartnershipByProjectAndPartnerLoader)
+    loader: LoaderOf<PartnershipByProjectAndPartnerLoader>,
+  ): Promise<Partnership> {
+    const result = await loader.load({
+      project: project.id,
+      partner: partnerId,
+    });
+    return result.partnership;
```

Comment:
@CarsonF we see the additional `id` property coming back which contains the project id and the partner id, but we only send back results.partnership.  Should we revise so that we are sending both the `id` and `partnership` objects back to the front end?

Reply:
I don't understand? are you getting a schema error?

Reply:
No error, we see the partnership object coming through on the front end, but wasn't sure in your design plans whether you wanted that id object also passed back up to the front end.

Reply:
ok